dist/angular-google-maps_dev_mapped.js

```diff
@@ -6137,7 +6137,11 @@ Original idea from: http://stackoverflow.com/questions/22758950/google-map-drawi
                 zoom: scope.zoom,
                 bounds: scope.bounds
               });
-              _gMap = new google.maps.Map(el.find('div')[1], mapOptions);
+              if (typeof geeCreateFusionMap === 'function') {
+                _gMap = new geeCreateFusionMap("angular-google-map-container", geeServerDefs, mapOptions);
```

Comment:
geeServerDefs is a global var created by a request to GEE, as described in the link from the PR:
```
<script type="text/javascript" src="/YourPublishedDatabase/query?request=Json&var=geeServerDefs"></script>
```